facade/service.go

```diff
@@ -324,7 +324,16 @@ func (f *Facade) GetServiceEndpoints(ctx datastore.Context, serviceId string) (m
 func (f *Facade) FindChildService(ctx datastore.Context, serviceId string, childName string) (*service.Service, error) {
 	glog.V(3).Infof("Facade.FindChildService")
 	store := f.serviceStore
-	return store.FindChildService(ctx, serviceId, childName)
+	parentService, err := store.Get(ctx, serviceId)
```

Comment:
is parentService the parent of the service with ID == serviceId? Or is it the exact service with ID == serviceId?